lib/utils.js

```diff
@@ -0,0 +1,173 @@
+import fs from 'fs/promises';
+import path from 'path';
+import Ajv from 'ajv/dist/2020';
+import {SSM} from '@aws-sdk/client-ssm';
+import got from 'got';
+import {Sequelize} from 'sequelize';
+import initModels from '@podaac/swodlr-db-models';
+import winston from 'winston';
+
+
+/**
+ * A class which manages automagic initialization of required services and
+ * singletons of common tools. Mainly accessible by an exported singleton
+ */
+class Utilities {
+  static #APP_NAME = 'swodlr';
+  static #SERVICE_NAME = 'async-update';
+  static #SSM_PATH = `/service/${this.#APP_NAME}/${this.#SERVICE_NAME}/`;
+  static #instance;
+
+  #ajv = new Ajv({
+    useDefaults: true,
+    removeAdditional: true,
+    loadSchema: async (uri) => await got(uri).json(),
+  });
+  #ssm = new SSM();
```

Comment:
Webstorm seems to think the SSM constructor requires 1 argument
> Invalid number of arguments, expected 1

Reply:
Should work without one - https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SSM.html

Reply:
Updated in [ef3089c](https://github.com/podaac/swodlr-async-update/pull/1/commits/ef3089ce65c42389e83b329a3e25cbfe1ccc1688)